### DISCUSSION TITLE
NextJS Expert Developers For Contract Hire?

### DISCUSSION THREAD

**Discussion by e-karma** [vercel/next.js#11556] [Help]
Whats the best place to find NextJS Experts interested in a few hours a week of project assistance? 

We have development capabilities, recently transitioned to NextJS and are pushing the boundaries of whats possible and are looking to accelerate progress.  In short, a high hourly rate will be offered for efficient, on-demand bottleneck assistance as problems emerge.

If anyone reading this is interested, let me know!

**Comment by kvnwolf** [comment]
Hi, @e-karma. I am interested. How can we talk?

**Comment by codinronan** [comment]
@e-karma would love to talk. We (I) have built websites handling over 150 million unique views a month (billions of total views a month -- very high traffic) moving hundreds of millions of dollars using next.js, and can share which in privat. This is my expertise and I'm sure I can help accelerate your development.

Since Github's contact options are limited feel free to reach out at patrick@engineerapart.com

**Comment by iaincollins** [comment]
If anyone needs help (either from an individual developer/contributor or as an agency) I'm currently avalible and happy to help with pretty much any aspect of development or deployment.

I've used it on dozens of commerical and open source projects; highlights include for the new The Economist website, for Amnesty International, an audio/video translation system funded by Google.

If I can help, you can drop me line at iain.collins@glitch.digital!

\* _We are based in London, but we are a UK company (VAT registered) and can and have worked with orgs in the EU, US and elsewhere._

PS: I would be remiss if I didn't also suggest getting in touch with someone at Zeit (am sure @timneutkens can forward you to the right person!) as I think they offer support contracts and supporting Next.js is🔥🔥🔥
